docs/examples/tests/guides/test_estimators_guide.py

```diff
@@ -0,0 +1,3 @@
+from ...guides.estimators_guide import EstimateExpectationValues
+
+"""this test file exists only to run the code in the tutorial files"""
```

Comment:
I don't think we need these files at all? 
1. It just imports, does not run anything
2. We can have one file which just executes all the scripts that we want to test, right?

Reply:
Importing the file runs all the code snippets right? If it was not imported, then the snippets would not be run with the tests.